### ISSUE TITLE
Registry Review

### ISSUE THREAD

**Issue by ngokevin** [fernandojsg/aframe-uploadcare-component#2]
- [x] Add image to README, like UploadCare logo.
- [ ] Add links to demo on GitHub README + description.
- [ ] Events no hyphen and present tense to be consistent with A-Frame.

**Comment by ngokevin** [comment]
Bump

**Comment by fernandojsg** [comment]
@ngokevin regarding the events, do you mean:
`uploadcare-upload-started => uploadcare-upload-start`
`uploadcare-upload-completed => uploadcare-upload-complete` ?

About the example, right now I didn't add an example just not to publish a valid uploadcare key, maybe should I just add an example showing the code but without a valid key? or create a new key in our uploadcare account with some expire date and use that?

**Comment by ngokevin** [comment]
Yes, also no dashes/hyphens. I guess no example is fine. In place, you can configure GitHub Pages + a theme to show the README with styling :P

**Comment by fernandojsg** [comment]
Ok, and what about remove the `uploadcare-` prefix too? as removing the dashes make it hard to read `uploadcareuploadstart` => `uploadstart`

**Comment by ngokevin** [comment]
Yeah `uploadstart` sounds fine
